ip-handling/draft-ietf-rtcweb-ip-handling.xml

```diff
@@ -127,11 +127,22 @@ ipr="trust200902">
       corporate VPN users) may in fact prefer WebRTC to send media traffic
       directly, i.e., not through the VPN.</t>
 
-      <t>#2 is considered to be a less significant concern, given that the
-      local address values often contain minimal information (e.g.,
-      192.168.0.2), or have built-in privacy protection (e.g., the
-      <xref target="RFC4941" /> IPv6 addresses recommended
-      by <xref target="I-D.ietf-rtcweb-transports"/>).</t>
+      <t>#2, is a less significant but valid concern. While the
+      <xref target="RFC4941" /> IPv6 addresses
+      recommended by <xref target="I-D.ietf-rtcweb-transports"/> are fairly
+      benign due to their intentionally short lifetimes, IPv4 addresses
+      present some challenges. Although they typically contain minimal entropy
+      (e.g., 192.168.0.2, a fairly common address), in the worst case,
+      they can contain 24 bits of entropy with an indefinite lifetime. As such,
+      they can be a fairly significant fingerprinting surface. In addition,
+      intranet web sites can be attacked
+      more easily when their IPv4 address range is externally known.</t>
+
+      <t>Private IP addresses can also act as an identifier that allows
+      web applications running in isolated browsing contexts (e.g., normal and
+      private browsing) to learn that they are running on the same device.
+      This could allow the application sessions to be correlated, defeating
+      some of the privacy protections provided by isolation.</t>
```

Comment:
While it is of course not wrong that it would help in gathering this information, during the F2F in Stockholm someone pointed out that there are many other ways to gather this information, too. IIRC it was @alvestrand.

RTT, memory pressure, ...

Reply:
There are indeed existing mechanism like memory pressure. This is not tied with WebRTC so is out of scope of this particular document.

IP addresses and RTT are issues identified in the MDNS ICE candidate draft that are specific to WebRTC. It might be good to mention both IP addresses and RTT.

Reply:
I updated the text to note that there are other potential mechanisms.

I chose not to mention RTT because it's not obvious from this document why one would need to use RTT - if you have the naked IP address, you can just strcmp it. My suggestion is just to introduce the correlation problem here, and then we can refer to it in the mDNS document to mention that even if the address is obscured, you can still perform correlation via RTT, and then discuss the implications.